docs/migration/2_0.md

```diff
@@ -10,7 +10,11 @@ Before 2.0 some dependencies of the `CmsPageGuard` were not public, so the guard
 
 Previously config validator logic was part of `ConfigModule`. If you are not using `StorefrontFoundationModule` or any of its descendants, it's required to import `ConfigValidatorModule.forRoot()` in order to make config validators run.
 
-### Factories removed from storefrontlib public API
+### Selectors removed from @spartacus/core
+
+`getSelectedProductsFactory` was removed as it was outdated.
```

Comment:
```suggestion
`ProductSelectors.getSelectedProductsFactory` was removed as it was outdated.
```

Maybe let's present it that way, as it was namespaced from lib consumers perspective?